Tools/WinMLRunner/WinMLRunner.sln

```diff
@@ -57,6 +59,16 @@ Global
 		{C3BCBEA1-90E6-426F-88AC-64C274BCEF45}.Release|x64.Build.0 = Release|x64
 		{C3BCBEA1-90E6-426F-88AC-64C274BCEF45}.Release|x86.ActiveCfg = Release|Win32
 		{C3BCBEA1-90E6-426F-88AC-64C274BCEF45}.Release|x86.Build.0 = Release|Win32
+		{C174D45D-C189-475B-B1A7-494939EE7491}.Debug|ARM64.ActiveCfg = Debug|Win32
```

Comment:
This will build Win32 binary when ARM64 is specified and static linking will be throw errors

Reply:
Hmm I wonder why this is happening with visual studio :(